plugin-gestures/src/test/java/com/mapbox/maps/plugin/gestures/GesturesPluginTest.kt

```diff
@@ -486,7 +488,8 @@ class GesturesPluginTest {
   }
 
   @Test
-  fun verifyScale() {
+  fun verifyScaleWithHighLevelAnimation() {
+    presenter.updateSettings { simultaneousRotateAndPinchToZoomEnabled = false }
```

Comment:
let's rename test to identify that in this case `simultaneousRotateAndPinchToZoomEnabled = false`